src/content/en/2019/javascript.md

```diff
@@ -0,0 +1,338 @@
+---
+part_number: II
+chapter_number: 1
+title: JavaScript
+authors: [housseindjirdeh]
+reviewers: [rviscomi, obto, paulcalvano, mathiasbynens]
+---
+
+# Chapter 1: JavaScript
```

Comment:
Should we add the hero image ourselves?

Reply:
No need, hero image and chapter/title info get added dynamically. Feel free to leave everything as-is and I'll clean it up during editing.